test.js

```diff
@@ -1,5 +1,5 @@
 var t = "hello";
 
-if (t === "hello') {
-    console.log('ok');
+if (t === "hello") {
+    var OK = 'OKAYY';
```

Comment:
Reporter: CHECKSTYLE
Rule: eslint.rules.no-unused-vars
Severity: ERROR
File: /var/lib/openshift/57246f9589f5cf681e000139/app-root/data/jobs/QA-PR-CHECK/workspace/test.js L4
 

&apos;OK&apos; is defined but never used (no-unused-vars)

<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>